### PR TITLE
Swope ocp baremetal

### PR DESCRIPTION
**Pull request by swopebe**
The memory and such was added because it is in the others, but without confirmation, I will need to remove. Let me know. Thanks!

### CODE REVIEW

File: install/plan_capacity.adoc [hunk 1]
```diff
@@ -27,7 +27,7 @@ Each {product-title} cluster is unique and the following guidelines provide samp
 In addition to {product-title-short}, the {ocp-short} cluster runs additional services to support cluster features. The following node sizes (3 nodes of the types noted in the information that follows, distributed evenly across 3 availability zones) are recommended.
 
 [#example-1k-sno]
-=== Example scenario: Creating and managing 1000 single node OpenShift clusters
+=== Example scenario: Creating and managing 1000 single node {ocp-short} clusters
```
Comment (berenss) [inline install/plan_capacity.adoc]: We support 2000 now, verified with ACM 2.4 code

File: install/plan_capacity.adoc [hunk 2]
```diff
@@ -128,3 +128,16 @@ IBM Power systems provide the ability to configure simultaneous multithreading (
 +
 For more information about SMT, see https://www.ibm.com/docs/en/aix/7.2?topic=concepts-simultaneous-multithreading[Simultaneous multithreading].
 
+[#ocp-bare-metal]
+=== {ocp-short} cluster on bare metal assets
```
Comment (berenss) [inline install/plan_capacity.adoc:132]: I would just prefer to call these bare metal hosts. or bare metal.

File: install/plan_capacity.adoc [hunk 3]
```diff
@@ -128,3 +128,16 @@ IBM Power systems provide the ability to configure simultaneous multithreading (
...
+
+See the following https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html#installation-three-node-cluster_installing-restricted-networks-bare-metal[product documentation] for more details.
+
+A {product-title} hub cluster can be installed and supported on {ocp-short} bare metal. The hub cluster can run on a compact bare metal topology, in which there are 3 schedulable control plane nodes, and 0 additional workers.
+
```
Comment (berenss) [inline install/plan_capacity.adoc:137]: excellent!

**Review by berenss — APPROVED** [review]
thank you!

File: install/plan_capacity.adoc [hunk 4]
```diff
@@ -128,3 +128,16 @@ IBM Power systems provide the ability to configure simultaneous multithreading (
...
+ * Node count: 3
```
Comment (swopebe) [inline install/plan_capacity.adoc:138]: @ch-stark are you able to test this section?

Comment (berenss) [reply install/plan_capacity.adoc:138]: this is being actively used @schmaustech has it running right now on 2.4.2

Comment (swopebe) [reply install/plan_capacity.adoc:138]: Does the entire paragraph work?

Comment (swopebe) [reply install/plan_capacity.adoc:138]: * Availability zones: 3

 * Instance size: 

 ** Memory: 16 GB

 ** Storage size: 120 GB

 ** vCPUs: 4

**Review by ch-stark — COMMENTED** [review]
looks fine for me if @schmaustech could also comment would be great

File: install/plan_capacity.adoc (hunk 4 above)
Comment (swopebe) [reply install/plan_capacity.adoc:138]: @schmaustech I will have to merge this soon, will you take a look? Thanks so much.

**Pull request merged**